### DISCUSSION TITLE
new events

### DISCUSSION THREAD

**Discussion by iniga72** [Zrips/CMI#8935] [Feature Requests]
please add events
 /mute 

and an event in the chat. where can I get and change the text of the message and the list of players who will receive the message

**Comment by CloudeLecaw** [comment]
what should this event done?
mute a player? a command for that exist...

or you want eventcommand events when player is muted?

or a list of comand that muted users can not use?
or a list of commands you can define for mute and unmute? (we do this by customalias allready...)
...

**Reply by iniga72** [reply]
the kick analog

public void CMIPlayerKickEvent(CMIPlayerKickEvent e){
Player p = Bukkit.getPlayer(e.getBanned());
e.setCancelled(true);
e.getBannedBy().sendMessage("no");
}

**Comment by iniga72** [comment]
please add CMIPlayerMuteEvent
